### ISSUE TITLE
integer constraint

### ISSUE THREAD

**Issue by karlseguin** [PrajwalCH/yazap#11]
Thanks for the library.

Any chance to add something like `singleValueOptionInt(T)` which parses the input into an int of type T (making sure both that it's a valid integer and that it's within min(T) and max(T).

**Comment by PrajwalCH** [comment]
Yes, it is possible to add something like that. In fact, I am planning to add support for parsing values into other types as well. Once i have some free time, i will start to implementing it.
